home/views.py

```diff
@@ -0,0 +1,33 @@
+from django.shortcuts import render, redirect
+from django.http import HttpResponse
+from django.contrib.auth.decorators import login_required
+from .models import Users
+
+@login_required
+def home(request):
+	if request.method == 'POST':
```

Comment:
It looks like this file is using tabs for indentation, the Python standard is typically to use 4 space indentation. You can make it easier to get this right in all of the files with an [.editorconfig](https://editorconfig.org) file. Most code editors and IDEs support this (but maybe with an extension).